### PR TITLE
add shareable link option

### PR DESCRIPTION
**Pull request by yi-fang-yuan**
# Summary

Adding shareable link feature  to each mural detail. Now  we can just click on get shareable link to have the url with the mural id added to the clipboard.
image

Also, made sure that the popup does work once we paste in the corresponding mural id 

image

## Test Plan

How did you test your changes?

By copy pasting the url lol
## Related Issues

#176

### CODE REVIEW

File: frontend/src/App/App.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ function App() {
   const [JWTtoken, setJWTtoken] = useState<any>();
 
   const [sidebarOpen, setSidebarOpen] = useState<boolean>(false);
-  const [activeForm, setActiveForm] = useState<FORM>(FORM.MURAL);
+  const [activeForm,  setActiveForm] = useState<FORM>(FORM.MURAL);
```
Comment (celinehuang) [inline frontend/src/App/App.tsx:30]: extra space here

File: frontend/src/muralForm/MuralForm.tsx [hunk 2]
```diff
@@ -271,6 +272,41 @@ function MuralForm({ mural, handleCancel }: IMuralFormProps) {
     return null;
   }
 
+  function copy() {
```
Comment (celinehuang) [inline frontend/src/muralForm/MuralForm.tsx:275]: maybe name this function name a bit more specific so there's better readability? nitpick tho

File: frontend/src/muralForm/MuralForm.tsx [hunk 3]
```diff
@@ -271,6 +272,41 @@ function MuralForm({ mural, handleCancel }: IMuralFormProps) {
...
+    return (
+      <Button
+      color="primary"
+      size="medium"
+      variant="outlined"
+      disableElevation
+      className={styles.directionButton}
+      onClick={copy}
+      >
+        <ShareIcon >
```
Comment (celinehuang) [inline frontend/src/muralForm/MuralForm.tsx:304]: extra space here as well

File: frontend/src/muralForm/MuralForm.tsx (hunk 2 above)
Comment (yi-fang-yuan) [reply frontend/src/muralForm/MuralForm.tsx]: changed

File: frontend/src/muralForm/MuralForm.tsx (hunk 3 above)
Comment (yi-fang-yuan) [reply frontend/src/muralForm/MuralForm.tsx]: changed

**Comment by tedspare** [comment]
Adjusted button styling to match "Directions"

File: frontend/src/App/App.tsx [hunk 4]
```diff
@@ -201,6 +201,21 @@ function App() {
...
+    setSelectedResource(data);
+  };
+
+  useEffect(() => {
+    if (window.location.pathname !== "/") {
+      setActiveForm(FORM.MURAL);
+      getSpecificMural();
+      setWelcomeOpen(false);
+    }
+  }, []);
```
Comment (tedspare) [inline frontend/src/App/App.tsx:217]: Clean solution

**Review by tedspare — APPROVED** [review]
LGTM

**Pull request merged**